sandbox-backend/src/main/java/com/exadel/sandbox/team5/mapper/MapperDTO.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.sandbox.team5.mapper;
+
+import com.exadel.sandbox.team5.dto.DiscountDTO;
+import com.exadel.sandbox.team5.entity.Discount;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MapperDTO {
+
+    private final ModelMapper modelMapper;
+
+    public DiscountDTO mapToDiscountDto(Discount discount) {
```

Comment:
Using of generic class can help to avoid dependence on each entity class. We can use for example the following construction to define entity class from type arguments in the constructor, for example see the following https://stackoverflow.com/questions/1901164/get-type-of-a-generic-parameter-in-java-with-reflection